<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSaveVolume</class>
 <widget class="QDialog" name="DialogSaveVolume">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Volume As</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditFileName">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOpen">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="freeview.qrc">
         <normaloff>:/resource/icons/file_open_16.png</normaloff>:/resource/icons/file_open_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxNoResample">
     <property name="text">
      <string>Do not resample when saving</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxCrop">
     <property name="text">
      <string>Crop to original volume size</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSaveVolume</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpen</sender>
   <signal>clicked()</signal>
   <receiver>DialogSaveVolume</receiver>
   <slot>OnOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSaveVolume</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxNoResample</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxCrop</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnOpen()</slot>
  <slot>OnOK()</slot>
 </slots>
</ui>
